src/main/webapp/app/exam/manage/students/upload-images/students-upload-images-dialog.component.scss

```diff
@@ -0,0 +1,107 @@
+$success-text-color: #155724;
+$success-bg-color: #d4edda;
+
+$error-text-color: #dc3545;
+$error-bg-color: #f8d7da;
+
+:host {
+    display: block;
+}
+
+.loading-spinner {
+    position: relative;
+    top: -2px;
+}
+
+.label-narrow {
+    display: inline-block;
+    margin-bottom: 0.1rem;
+}
+
+.import-success,
+.import-fail {
+    position: relative;
+
+    &:nth-child(odd):before {
+        content: '';
+        pointer-events: none;
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        background: rgba(0, 0, 0, 0.03);
+    }
+}
+
+table.table.table-striped .import-success {
```

Comment:
This and the following definition can be nested for better readability.

Reply:
@pellumbbaboci please add this in a follow-up PR. You can contact @bassner if you need help!